drivers/spi/Kconfig.nrfx

```diff
@@ -206,6 +206,52 @@ config SPI_NRFX_RAM_BUFFER_SIZE
 	  supplying buffers located in flash to the driver, otherwise such
 	  transfers will fail.
 
+choice
+	prompt "nRF SPIM MISO pin pull configuration"
+	default SPI_NRFX_SPIM_PULL_DOWN
+
+config SPI_NRFX_SPIM_NO_PULL
+	bool "no pull"
+	help
+	  Disable MISO pin pull.
+
+config SPI_NRFX_SPIM_PULL_DOWN
+	bool "pull down"
+	help
+	  Enable MISO pin pull down.
+
+config SPI_NRFX_SPIM_PULL_UP
+	bool "pull up"
+	help
+	  Enable MISO pin pull up.
+
+endchoice
+
 endif # NRFX_SPIM
 
+if NRFX_SPI
+
+choice
+	prompt "nRF SPI MISO pin pull configuration"
```

Comment:
I am not convinced that we should provide this choice separately for SPI and SPIM. It should be configurable for each instance independently, but this is not currently supported by nrfx. So I think for now we could go with a common choice option and just indicate in the prompt that it is not supported for SPIS ("SPI/SPIM MISO pin pull configuration" maybe?). And the choices should have `MISO` in their names, instead of `SPI` or `SPIM`.

Reply:
nrfx has independent config options for SPI and SPIM, this was the reason behind having two options here. If these can be used together (e.g. one for SPI0 while the other for SPI1) I thing it would be better to leave as is (allow user to use functionality available in nrfx). If you think these two (SPI and SPIM) cannot be used independently then indeed one choice is enough.

Reply:
Yes, you could use e.g. SPI0 and SPIM1, and in such case you could then configure pulls for MISO pins independently. I just thought this might be confusing for users. But okay. Let's keep these two options. At least this will be a way (not ideal but still) to achieve different MISO pull configurations for different instances.
But regarding the naming, I still think it would be good to include `MISO` in the names of these choices elements. `SPI_NRFX_SPIM_PULL_DOWN=y` might look ambiguous in the application configuration.

Reply:
thanks. Would you like to simply remove MISO from the name or use some other name?

Reply:
I thought about just adding `MISO` to the names.
For example, to use `config SPI_NRFX_SPIM_MISO_PULL_DOWN` and so on.

Reply:
ah, ok. I will add it and push the change.